src/qibolab/_core/instruments/qblox/config.py

```diff
@@ -0,0 +1,153 @@
+import json
+from typing import Optional, cast
+
+from qblox_instruments.qcodes_drivers.module import Module
+from qblox_instruments.qcodes_drivers.sequencer import Sequencer
+
+from qibolab._core.components.channels import AcquisitionChannel, Channel, IqChannel
+from qibolab._core.components.configs import (
+    AcquisitionConfig,
+    Configs,
+    DcConfig,
+    IqConfig,
+    OscillatorConfig,
+)
+from qibolab._core.execution_parameters import AcquisitionType
+from qibolab._core.identifier import ChannelId
+from qibolab._core.serialize import Model
+
+from .identifiers import SlotId
+from .sequence import Q1Sequence
+
+__all__ = []
+
+
+class PortAddress(Model):
+    slot: SlotId
+    ports: tuple[int, Optional[int]]
+    input: bool = False
+
+    @classmethod
+    def from_path(cls, path: str):
+        """Load address from :attr:`qibolab.Channel.path`."""
+        els = path.split("/")
+        assert len(els) == 2
+        ports = els[1][1:].split("_")
+        assert 1 <= len(ports) <= 2
+        return cls(
+            slot=int(els[0]),
+            ports=(int(ports[0]), int(ports[1]) if len(ports) == 2 else None),
+            input=els[1][0] == "i",
+        )
+
+    @property
+    def local_address(self):
+        """Physical address within the module.
+
+        It will generate a string in the format ``<direction><channel>`` or
+        ``<direction><I-channel>_<Q-channel>``.
```

Comment:
Is there a practical case where this applies (basically where `self.ports[1] is not None`)? 

I was expecting for RF channels, however for both the drive and probe/acquisition of the iqm5q all `ports` are single integer and the second element of the tuple is always `None`, so I am wondering if that is correct or there is something wrong with the platform definition or `PortAddress` creation.

Reply:
> Is there a practical case where this applies (basically where `self.ports[1] is not None`)?

Not for the time being.

I believe the idea to be that you may use a single sequencer to drive two ports, because the two will actually correspond to IQ components.
In which case, it may be useful if you're using the QCM (not RF) in order to generate an RF signal, upconverting it with an external LO. This is not currently happening in any of ours platforms...